gadgets/trace_lsm/gadget.yaml

```
@@ -0,0 +1,638 @@
+name: trace lsm
+description: a strace for LSM tracepoints
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://inspektor-gadget.io/docs
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+tracers:
+  lsm:
+    mapName: events
+    structName: event
+structs:
+  event:
+    fields:
+    - name: mntns_id
+      description: Mount namespace inode id
+      attributes:
+        template: ns
+    - name: pid
+      attributes:
+        template: pid
+    - name: tid
+      attributes:
+        template: pid
+    - name: uid
+      attributes:
+        template: pid
+    - name: gid
+      attributes:
+        template: pid
+    - name: tracepoint
+      description: lsm tracepoint enum
+      attributes:
+        width: 16
+        alignment: left
+        ellipsis: end
+    - name: comm
+      description: command
+      attributes:
+        template: comm
+    - name: timestamp
+      attributes:
+        template: timestamp
+ebpfParams:
+  trace_all:
+    key: trace_all
```

```suggestion
    key: trace-all
```

Other gadgets use a `-` for the CLI flag:
```bash
$ git grep 'key: .*[-_].*' -- gadgets/
gadgets/top_file/gadget.yaml:58:    key: all-files
gadgets/trace_exec/gadget.yaml:84:    key: ignore-failed
gadgets/trace_signal/gadget.yaml:51:    key: kill-only
```